gap/attr.gi

```diff
@@ -1582,3 +1582,125 @@ function(gr)
   od;
   return fail;
 end);
+
+InstallMethod(VertexConnectivity,
+"for a digraph",
+[IsDigraph],
+function(digraph)
+  local edges, edmondskarp, k, kappa, kappas, mindegv, Nv, outdegs, relabel,
+  sdigraph, set1v, x, y;
+
+  if DigraphNrVertices(digraph) = 0 then
+    return 0;
+  fi;
+
+  sdigraph := DigraphSymmetricClosure(DigraphRemoveAllMultipleEdges(digraph));
+  edges := DigraphEdges(sdigraph);
+  kappas := [DigraphNrVertices(digraph) - 1];
+
+  relabel := function(edge, source, sink, wvert)
+    local nedge, i;
+    nedge := [];
+
+    for i in [1, 2] do
+      if edge[i] = source then
+        nedge[i] := 1;
+      elif edge[i] = sink then
+        nedge[i] := 2;
+      else
+        nedge[i] := Position(wvert, edge[i]) * 2 + i;
+      fi;
+    od;
+    return nedge;
+  end;
+
+  kappa := function(source, sink, digraph, edges)
+    local wvert, nedges, i;
+    wvert := ShallowCopy(DigraphVertices(digraph));
+    Remove(wvert, PositionSorted(wvert, source));
+    Remove(wvert, PositionSorted(wvert, sink));
+    nedges := List(edges, x -> relabel(x, source, sink, wvert));
+
+    for i in [1 .. Size(wvert)] do
+      Add(nedges, [i * 2 + 2, i * 2 + 1]);
+    od;
+
+    return edmondskarp(DigraphByEdges(nedges), 1, 2);
+  end;
+
+# The following function is an implementation of the Edmonds-Karp algorithm with
+# some minor adjustments that take into account the fact that the capacity of
+# all edges is 1.
+  edmondskarp := function(network, source, sink)
+    local flow, predecessor, queue, current, flag, e, capacity, outn, n;
+    flag := true;
+    flow := 0;
+    outn := OutNeighbours(network);
+    capacity := List(outn, x -> List(x, y -> true));
+
+    while flag do
+      queue := [source];
+      predecessor := List(outn, x -> 0);
+
+      while not IsEmpty(queue) do
+        current := queue[1];
+        Remove(queue, 1);
```

Comment:
This is probably quite costly in GAP, better not to use a queue of this type. Please refactor this to remove `Remove`